src/screens/Customers/components/CustomerCard.tsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import { createStyles, Theme, makeStyles } from '@material-ui/core/styles';
+import { Card, CardActions, Typography, IconButton } from '@material-ui/core';
+import ArrowForwardIosIcon from '@material-ui/icons/ArrowForwardIos';
+import AttachMoneyIcon from '@material-ui/icons/AttachMoney';
+import FlashOnIcon from '@material-ui/icons/FlashOn';
+import { Link } from 'react-router-dom';
+import { CustomerRecord } from '../../../lib/airtable/interface';
+import { setCurrentCustomerIdInRedux, CustomerStatus } from '../../../lib/redux/customerData';
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    cardContent: {
+      flexGrow: 1,
+      display: 'flex',
+      alignItems: 'center',
+      paddingLeft: '17px',
+      color: theme.palette.text.primary,
+    },
+    totalText: {
+      color: theme.palette.text.secondary,
+    },
+    numberText: {
+      color: theme.palette.error.main,
+      marginLeft: '5px',
+    },
+    divSpacing: {
+      paddingLeft: '15px',
+    },
+    singleCard: {
+      width: '100%',
+      height: '93px',
+      border: `1px solid ${theme.palette.text.disabled}`,
+      borderRadius: '8px',
+      textAlign: 'left',
+      display: 'flex',
+      flexDirection: 'row',
+      marginBottom: '10px',
+    },
```

Comment:
If you switch over to using `CardContent`, most of this will be unnecessary

Reply:
I don't know how I should be using CardContent to make things work, can you please show me an example?

Reply:
Idk how to used CardContent with indicator, also I tried taking out some of the values with CardContent and then the righthand side broke

Reply:
Maybe we could add a TODO to consider this and file a github issue so it doesn't get lost? Just so it's in for bug bash.